### DISCUSSION TITLE
In earlier versions of Catalyst, running the npm create command would generate only a Next.js project by default but now the entire monorepo

### DISCUSSION THREAD

**Discussion by prasanthpillai** [bigcommerce/catalyst#1711] [Q&A]
In earlier versions of Catalyst, running the `npm create @bigcommerce/catalyst@latest` command would generate only a standalone Next.js project by default. However, it now generates the entire monorepo structure. Is this the expected behavior going forward?

**Comment by chanceaclark** [comment]
Hey @prasanthpillai, yeah this is the expected workflow. We mentioned changing over to this in https://github.com/bigcommerce/catalyst/discussions/1430

If you have any feedback around this workflow, would you mind dropping a comment in that discussion? Going to close this discussion to consolidate it over to that one.
